Calculator/InvestmentCalculator.csproj

```diff
@@ -43,7 +43,7 @@
     <Reference Include="System.Xml" />
   </ItemGroup>
   <ItemGroup>
-    <Compile Include="Program.cs" />
+    <Compile Include="InvestmentCalculator.cs" />
```

Comment:
It's Ok to rename the Program class for simple projects with a single or few classes. For a big project, it's better to have an entry point in the Program class because then it will be easier to understand where to start looking for an entry point